gramps/gen/datehandler/_date_nl.py

```diff
@@ -147,17 +149,17 @@ class DateDisplayNL(DateDisplay):
 
     calendar = (
         "", "juliaans", "hebreeuws",
-        "franse republiek", "persisch", "islamitisch",
+        "frans republikeins", "persisch", "islamitisch",
         "zweeds" )
 
-    _mod_str = ("", "voor ", "na ", "rond ", "", "", "")
+    _mod_str = ("", "voor ", "na ", "omstreeks ", "", "", "")
```

Comment:
Either word would work here :)

Reply:
Not any more, in new Dutch translation I have changed 'rond' into 'omstreeks'. So datehandler should contain 'omstreeks' too. Otherwise you could get an error when trying to fill in 'omstreeks'.
Maybe it's possible to have both words?